src/braket/timings/time_series.py

```diff
@@ -87,6 +87,120 @@ def _ensure_sorted(self) -> None:
             self._series = OrderedDict(sorted(self._series.items()))
             self._sorted = True
 
+    @staticmethod
+    def from_lists(times: List[float], values: List[float]) -> TimeSeries:
+        """Create a time series from the list of time and value points
+
+        Args:
+            times (List[float]): list of time points
+            values (List[float]): list of value points
+
+        Returns:
+            TimeSeries: time series constructed from lists
+        """
+        if len(times) != len(values):
+            raise ValueError(
+                f"The lengths of the times({len(times)})\
+                      and values({len(values)}) lists are not equal."
+            )
+
+        ts = TimeSeries()
+        for t, v in zip(times, values):
+            ts.put(t, v)
+        return ts
+
+    @staticmethod
+    def constant_like(times: Union[List[float], TimeSeries], constant: float = 0.0) -> TimeSeries:
+        """Obtain a constant time series given the list of time points and the constant values
+
+        Args:
+            times (List[float]): list of time points
+
+        Returns:
+            TimeSeries: A constant time series
+        """
+        ts = TimeSeries()
+        for t in times:
+            ts.put(t, constant)
+        return ts
+
+    def concatenate(self, other: TimeSeries) -> TimeSeries:
+        """Concatenate two time series to a single time series
+
+        Args:
+            other (TimeSeries): The second time series to be concatenated
+
+        Returns:
+            TimeSeries: The concatenated time series.
+
+        Notes:
+            Keeps the time points in both time series unchanged.
+            Assumes that the time points in the first TimeSeries
+            are at earler times then the time points in the second TimeSeries.
+        """
+
+        if len(other.times()) * len(self.times()) == 0:
```

Comment:
Have you tested when one time series is empty, but the other is not? 

Regardless, can you rewrite this so the "pass" isn't necessary?

Reply:
thanks, rewrote the code without "pass".

Also added more test functionality when wither one or both TimeSeries are empty